src/interval.jl

```diff
@@ -170,24 +170,21 @@ end
 Base.isless(a::AbstractInterval, b) = LeftEndpoint(a) < b
 Base.isless(a, b::AbstractInterval) = a < LeftEndpoint(b)
 
-less_than_disjoint(a::AbstractInterval, b) = RightEndpoint(a) < b
-less_than_disjoint(a, b::AbstractInterval) = a < LeftEndpoint(b)
-
-function Base.:isless(a::AbstractInterval, b::AbstractInterval)
+function Base.isless(a::AbstractInterval, b::AbstractInterval)
     return LeftEndpoint(a) < LeftEndpoint(b)
 end
 
-function less_than_disjoint(a::AbstractInterval, b::AbstractInterval)
+isless_disjoint(a::AbstractInterval, b) = RightEndpoint(a) < b
```

Comment:
If it's called `isless`-something it should call `isless`, not `<`.

Reply:
Good point. I need to some investigation here.